src/components/main/compose/mod.rs

```diff
@@ -108,24 +108,24 @@ pub fn Compose(cx: Scope<Props>) -> Element {
                             }
 
                             // clicking the send button is meaningless if there isn't a conversation.
-                            if let Some(id) = current_chat {
+                            if let Some(convo) = current_chat.clone() {
```

Comment:
if `current_chat` gets changed back to a Uuid then the `clone()` won't be necessary. 

But, in the future, a way to avoid having to clone here is as follows: ` if let Some(convo) = &current_chat {`